sdk/core/core-client/src/serializationPolicy.ts

```diff
@@ -132,7 +132,11 @@ export function serializeRequestBody(
     const typeName = bodyMapper.type.name;
 
     try {
-      if (request.body || required) {
+      if (
+        (request.body !== undefined && request.body !== null) ||
+        (nullable && request.body === null) ||
```

Comment:
You need to deconstruct `nullable` from the mapper though. Current code doesn't do it.

Reply:
Ops, did the change inside nano and did not pay attention if this var is in scope. Fixed.